### ISSUE TITLE
Add major version tags

### ISSUE THREAD

**Issue by fwal** [stefanzweifel/git-auto-commit-action#77]
**Is your feature request related to a problem? Please describe.**
It would be nice if there where major tags like `v4`, `v3` etc so I don't have to update my workflow files for every release 🙂.

**Describe the solution you'd like**
Add major version tags like `v4` as described in
https://github.com/actions/toolkit/blob/master/docs/action-versioning.md

**Describe alternatives you've considered**
n/a
**Additional context**
n/a

**Comment by stefanzweifel** [comment]
Thanks for the link! 
Wasn't aware that such a great documentation around this problem existed.

Will create major version tags during the next few hours.

**Comment by fwal** [comment]
No problem! I wasn't aware either until a couple of days ago 😅

**Comment by stefanzweifel** [comment]
And done. I've created major versions for `v1`, `v2`, `v3` and `v4`. Each major version corresponds to the latest released version in their "major-range". When new minor versions are being released, the corresponding major version tag will be updated too.

I've also updated the documentation to now only include the major version in the examples.

Thanks for bringing this up Frederik! This was also something that really annoyed me in my own projects 😅
